ci/docs/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+export HOME=$WORKSPACE
+export DOCS_WORKSPACE=$WORKSPACE/docs
+export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
+
+export CUDA_REL=${CUDA_VERSION%.*}
+export CUDA_SHORT=${CUDA_REL//./}
+export CLX_HOME=$WORKSPACE/clx_build
+
+# Switch to project root; also root of repo checkout
+cd $WORKSPACE
+export GIT_DESCRIBE_TAG=`git describe --tags`
+export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Check versions..."
+python --version
+$CC --version
+$CXX --version
+conda list
+
+#clx source build
+git clone --single-branch --branch branch-${BRANCH_VERSION} https://github.com/rapidsai/clx.git ${CLX_HOME}
+${CLX_HOME}/build.sh clean libclx clx
```

Comment:
Discussed this with AJ as well. This block should be removed. If this block exists in a script in this repo, then this repo shouldn't have to be cloned again. This should be added as an extra step in the `clx` docs build since it is not part of our runtime containers.